torch/csrc/autograd/python_variable.cpp

```diff
@@ -159,6 +159,19 @@ static PyObject* THPVariable_make_subclass(PyObject* _ignored, PyObject* args, P
 typedef PyObject *(*getter)(PyObject *, void *);
 typedef int (*setter)(PyObject *, PyObject *, void *);
 
+PyObject *THPVariable_get_T(THPVariable *self)
```

Comment:
We don't have the ability to do this in C++, yet?  Can we add support for it?

Reply:
Sure, what would you like to call it? `reverse_dims` ? Or just T?

Reply:
I'd suggest `T`. In general, we've tried to make the C++ Tensor API match the Python API even when it means uncommon names.